airbyte/mcp/cloud_ops.py

```diff
@@ -37,6 +33,7 @@ def _get_cloud_workspace() -> CloudWorkspace:
 
 
 # @app.tool()  # << deferred
+@cloud_tool(destructive=False)
```

Comment:
Maybe you are already working on this, but all of these refs should be modified to the equivalent of:

```
@mcp_tool(
    domain="cloud",
    destructive=False,
)
```

Use this multi-line format instead of the inline format. Also, remove the now-stale comments "# @app.tool()  # << deferred".